extensions/amp-story/1.0/test/test-media-performance-metrics-service.js

```diff
@@ -420,4 +420,39 @@ describes.fakeWin('media-performance-metrics-service', {amp: true}, (env) => {
       expect(tickStub).to.have.been.calledWithExactly('vcs', 2);
     });
   });
+
+  describe('Video on first page', () => {
+    it('should report the video is on the first page', () => {
+      const video = win.document.createElement('video');
+      const source = win.document.createElement('source');
+      const firstPage = win.document.createElement('amp-story-page');
+      source.setAttribute('src', 'foo.mp4');
+      video.appendChild(source);
+      firstPage.appendChild(video);
+      win.document.body.appendChild(firstPage);
+
+      service.startMeasuring(video);
+      service.stopMeasuring(video);
+
+      expect(tickStub).to.have.been.calledWithExactly('vofp', 1);
+    });
+
+    it('should report the video is not on the first page', () => {
```

Comment:
Sorry I know I'm being specific, but can you also add a video in the first page?

Reply:
Sure, as long as I don't call `startMeasure`